test/jdk/jdk/crac/ContextOrderTest.java

```diff
@@ -94,26 +95,26 @@ private static void testOrder() throws Exception {
         Core.checkpointRestore();
         assertTrue(recorder.stream().anyMatch("jdk-create-child2-before"::equals));
         assertTrue(recorder.stream().anyMatch("regular2-child2-before"::equals));
-
-        rememberMe.clear();
-        System.gc();
     }
 
     private static void testRegisterBlocks() throws Exception {
         var recorder = new LinkedList<String>();
+        BlockingOrderedContext<Resource> blockingCtx = new BlockingOrderedContext();
```

Comment:
We don't need separate contexts now that global one is blocking.

Reply:
The test checks blocking properties, while blocking behavior of the global context  is not specified.